src/javaRosa.js

```diff
@@ -1251,7 +1251,7 @@ define([
 
         function makeHeadings(languages, forms) {
             return rowify("label", languages, forms, function (language, form) {
-                return form + '-' + language;
+                return form + '_' + language;
```

Comment:
I think this will break imports. See [line 1205 in `parseXLSItext`](https://github.com/dimagi/Vellum/blob/export-trans/src/javaRosa.js#L1205) above.

Also, should probably make `parseXLSItext` handle both _ and - so that previously exported forms will still be importable. It seems to me that HQ's format was conceived after the form builder. Maybe HQ's import/export code would be a better place to fix this? (and I personally prefer - to _).